res/values/arrays.xml

```diff
@@ -52,6 +52,33 @@
     <item>Dvorak</item>
     <item>Colemak</item>
   </string-array>
+  <string-array name="pref_show_numpad_values">
+    <item>never</item>
+    <item>landscape</item>
+    <item>always</item>
+  </string-array>
+  <string-array name="pref_show_numpad_entries">
+    <item>@string/pref_show_numpad_never</item>
+    <item>@string/pref_show_numpad_landscape</item>
+    <item>@string/pref_show_numpad_always</item>
+  </string-array>
+  <string-array name="pref_numpad_layout_entries">
```

Comment:
I like customization but I think this is not the right way. The `pref_numeric` case is too crowded and doesn't look good, I don't want to think about that when modifying the numeric layout.

The period vs comma decimal separator is an interesting idea but I think it's not the best way to implement it. I'd like to have that on the regular numeric layout too. A better implementation would be a field in `method.xml` and then to perform arbitrary modifications like `Config.modify_layout`.

I suggest leaving this idea out of the PR for now. I'd prefer to have just one keypad that covers most cases (with the operators, if it's too wide, consider moving things into the corners of the digits).